lib/cryptoexchange/exchanges/cobinhood/services/pairs.rb

```diff
@@ -0,0 +1,31 @@
+module Cryptoexchange::Exchanges
+  module Cobinhood
+    module Services
+      class Pairs < Cryptoexchange::Services::Pairs
+        PAIRS_URL = "#{Cryptoexchange::Exchanges::Cobinhood::Market::API_URL}/market/trading_pairs"
+
+        def fetch_via_api(endpoint = self.class::PAIRS_URL)
+          ctx = OpenSSL::SSL::SSLContext.new
+          ctx.verify_mode = OpenSSL::SSL::VERIFY_NONE
```

Comment:
Is it possible to make do without SSL verify none?

Reply:
Right now, Cobinhood's SSL certificate is not valid.  The only way I found around it was to not verify it.  Perhaps merge for now, but keep an open issue ticket to check every 30 days or so so see if they've fixed their certificate issues?

Reply:
@mwlang Tested on my local machine without SSL seem to work fine, i'll accept this PR and remove the verify none SSL and see how this goes moving forward